lib/util/exec/index.ts

```diff
@@ -17,11 +17,14 @@ const execConfig: ExecConfig = {
   cacheDir: null,
 };
 
+const prefetchedDockerImages: Set<string> = new Set<string>();
+
 export function setExecConfig(config: Partial<RenovateConfig>): void {
   for (const key of Object.keys(execConfig)) {
     const value = config[key];
     execConfig[key] = value || null;
   }
+  prefetchedDockerImages.clear();
```

Comment:
Should be good enough for independent test contexts, as soon as we call `setExecConfig` once per run